pkg/catalog/template.go

```diff
@@ -3,7 +3,7 @@ package catalog
 
 const ProjectTemplate = `
 {{/* Automatically generated by https://github.com/anz-bank/sysl-catalog it is strongly recommended not to edit this file */}}
-{{range $name, $link := .Links}} [{{$name}}]({{$link}}) | {{end}} [Chat with us]({{.ChatLink}}) | [New bug or feature request]({{.FeedbackLink}})
+{{range $name, $link := .Links}} [{{$name}}]({{$link}}) | {{end}} | )
```

Comment:
Should we replace this with the default contact links? Since it'll be overwritten in custom templates
https://anzoss.slack.com/messages/sysl-catalog/ and
https://github.com/anz-bank/sysl-catalog/issues/new